src/components/RichTextViewer.tsx

```diff
@@ -10,6 +10,8 @@ const RichTextViewer = (props: { content?: string }) => {
 };
 
 const Wrapper = styled.div`
+	max-width: 750px;
+
```

Comment:
do you consider the responsiveness?

Reply:
Yes, this is just to limit on the large displays, but on the smaller ones everything is working fine for this element.